services/web/server/src/simcore_service_webserver/projects/_crud_api_create.py

```diff
@@ -320,6 +320,12 @@ async def create_project(  # pylint: disable=too-many-arguments,too-many-branche
                 # As user has access to the project, it has implicitly access to the folder
                 folder_id = prj_to_folder_db.folder_id
 
+            if as_template:
```

Comment:
THOUGHT: should we put some time into splitting the `Create` entrypoint?

Reply:
Yes! we need to. Also there are so many combinations that we do not have tests for most of them.